cluster-provision/k8s/1.24/k8s_provision.sh

```diff
@@ -55,9 +55,11 @@ cat <<EOF >/etc/NetworkManager/conf.d/002-dhclient.conf
 dhcp=dhclient
 EOF
 
-sysctl -w net.netfilter.nf_conntrack_max=1000000
```

Comment:
Can we leave a note about why do we make this change?

Reply:
Yes it happened again here on the cluster - https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirtci/800/check-provision-k8s-1.24/1531922243037171712#1:build-log.txt%3A13068 - I tried looking into this but I couldn't find any reason for it.

Reply:
I think that one of the packages we install creates it as well, so maybe the order changed in a subtle way causing it ?
maybe revert all the nice to have cosmetics, keep only the mandatory changes